vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -108,7 +109,12 @@ production values of companies to stay in line with carbon budgets.
 assumed to have. At this point we assume these are rolled over every year.
 * `scenario_geography` Indicates which geographical region(s) (concerning asset
 location) results shall be calculated for.
-* `use_company_terms` Logical parameter, indicating if individually provided
+* `company_exclusion` Logical parameter that indicates if certain combinations
```

Comment:
please remove any reference to company exclusion, it has been removed from the repo and now happens automatically